hack/jenkins/e2e.sh

```diff
@@ -164,6 +166,7 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
+    "GCE\sL7\sLoadBalancer\sController"
```

Comment:
They can't run in parallel with *themselves*, or other loadbalancer tests that use the same quota. I guess I was confused if this would ever happen. The second problem can be solved by simply requesting an increase (though oddly it looks like our slow-jenkins project is currently using 100% forwarding rules) as long as the first isn't a problem. Are you ok with bumping quotas way up (because we don't know how many L4 and L7 tests might run in parallel)?